### ISSUE TITLE
How to parallelize a model with fake quantization nodes?

### ISSUE THREAD

**Issue by sheegao** [intel/neural-compressor#1452]
If a quantized model contains fake quantization nodes, how can such a model be parallelized, and how can its accuracy be validated on a dataset?

**Comment by mengniwang95** [comment]
Hi @sheegao , which framework do you use?  Does 'fake quantization nodes' mean quant-dequant pairs?

**Comment by sheegao** [comment]
> Hi @sheegao , which framework do you use? Does 'fake quantization nodes' mean quant-dequant pairs?

Yes,it's mean quant-dequant pairs,but I have not found any existing distributed frameworks that support this type of model.

**Comment by xin3he** [comment]
Hi @sheegao , you can directly deploy a fake quantized torch model with torch [DistributedDataParallel API](https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html). I don't think this will be a problem.

**Comment by sheegao** [comment]
> Hi @sheegao , you can directly deploy a fake quantized torch model with torch [DistributedDataParallel API](https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html). I don't think this will be a problem.

yeah，But I'm interested in pipeline parallelism or tensor parallelism, rather than data parallelism

**Comment by xin3he** [comment]
I think this question should be raised to packages who provide pipeline parallelism or tensor parallelism.

**Comment by thuang6** [comment]
No further plan or discussion in this thread,  close it for now
